met-web/src/components/common/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { Button as MuiButton, Paper as MuiPaper, Grid, CircularProgress } from '@mui/material';
+import { Button as MuiButton, Container, Grid, CircularProgress } from '@mui/material';
 import styled from '@emotion/styled';
 
 export const RoundedButton = styled(MuiButton)(() => ({
     borderRadius: '23px',
 }));
 
-export const MetBox = styled(MuiPaper)(() => ({
-    border: `1px solid #606060`,
+export const MetBox = styled(Container)(() => ({
+    border: `1.5px solid #606060`,
```

Comment:
why change from 1px to 1.5px? 1px is as per the design https://xd.adobe.com/view/7360b654-4a25-4140-9670-a56f4adabe7a-2633/screen/32caf4d5-99c5-4df2-9e43-620e6b838203/specs/